planner/core/integration_test.go

```diff
@@ -1564,3 +1565,25 @@ func (s *testIntegrationSuite) TestDeleteUsingJoin(c *C) {
 	tk.MustQuery("select * from t1").Check(testkit.Rows("1 1"))
 	tk.MustQuery("select * from t2").Check(testkit.Rows("2 2"))
 }
+
+func (s *testIntegrationSerialSuite) Test19942(c *C) {
+	collate.SetNewCollationEnabledForTest(true)
+	defer collate.SetNewCollationEnabledForTest(false)
+
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("SET @@tidb_enable_clustered_index=1;")
```

Comment:
How about restoring the current `tidb_enable_clustered_index` here and restore it when the test is done?

Reply:
This is a session variable. The other tests use a different session by default.

Reply:
Oh I see, thanks.